onnxruntime/contrib_ops/cuda/bert/group_query_attention.cc

```diff
@@ -163,6 +163,10 @@ Status GroupQueryAttention<T>::ComputeInternal(OpKernelContext* context) const {
       parameters.sequence_length <= parameters.seqlen_past_kv_cache + parameters.sequence_length &&
       (sizeof(T) == 2 || parameters.sequence_length >= attention::kMinSeqLenForMemoryEfficientAttentionFp32) &&
       has_memory_efficient_attention(sm, sizeof(T) == 2);
+  if (!use_flash_attention && !use_memory_efficient_attention && local_window_size_ != -1) {
```

Comment:
>local_window_size_ [](http://example.com/codeflow?start=65&length=18)

can we just ignore the local_window_size_ and compute with full size in this case?